api/feature.go

```diff
@@ -13,8 +13,10 @@ func (f *Feature) UnmarshalText(text []byte) error {
 //go:generate enumer -type Feature
 const (
 	_ Feature = iota
-	Offline
-	CoarseCurrent
-	IntegratedDevice
-	Heating
+
+	Hidden           // vehicle: hidden from api
```

Comment:
Mir ist hier die Bedeutung von `hidden` noch nicht klar. Verstecken wir den auch im Ladelog? Soll das nur für die Fahrzeugselektion im Main UI gelten. Zeigen wir bei Hidden Fahrzeugen den SoC an aber nicht den Namen? Mir fehlt hier noch das klare Bild wann ein Nutzer genau diese Option aktivieren möchte.

Reply:
Können wir das Verstecken nicht einfach am Integrated Device festmachen und kommen zum gleiche Ergebnis? (Anwendungsfall @VolkerK62)

Reply:
>  Verstecken wir den auch im Ladelog?

Ladelog enthält ja auch den Charger? Da müsste es dann konsequenterweise raus.

> aben Hidden Fahrzeuge API Endpoints?

Eigentlich (tm) nicht.

> Mir fehlt hier noch das klare Bild wann ein Nutzer genau diese Option aktivieren möchte.

Müssen wir mal durchsprechen.

Reply:
> Müssen wir mal durchsprechen.

Sounds like a plan :D

Reply:
Ich fände es gut, wenn Hidden Fahrzeuge nur in den Auswahllisten versteckt würden, denn:
- Im Ladelog steht sonst bei "integrateddevice" Loadpoints immer "Gastfahrzeug"
- Im UI wird das Fahrzeug bei "integrateddevice" Loadpoints sowieso nicht angezeigt
Wobei ich persönlich es nach wie vor besser fände, stattdessen eine Funktion zur Zuordnung von Fahrzeugen zu Loadpoints zu machen, das wäre vorr. nur geringer Mehraufwand, würde den konkreten Fall auch erschlagen und böte mehr Möglichkeiten.